cypress/integration/v2/blocks/pricing-box.spec.js

```diff
@@ -59,6 +63,311 @@ function switchDesign() {
+			'padding-top': '25%',
+			'padding-bottom': '25%',
+			'padding-right': '25%',
+			'padding-left': '25%',
+		},
+	} )
+
+	cy.adjust( 'Image', 13, { viewport } )
+	cy.adjust( 'Title', 18, { viewport } )
+	cy.adjust( 'Price', 23, { viewport } )
+	cy.adjust( 'Sub Price', 28, { viewport } )
+	cy.adjust( 'Button', 33, { viewport } )
+	cy.adjust( 'Description', 38, { viewport } )
+		.assertComputedStyle( {
+			'.ugb-pricing-box__image': {
+				'margin-bottom': '13px',
+			},
+			'.ugb-pricing-box__title': {
+				'margin-bottom': '18px',
+			},
+			'.ugb-pricing-box__price-wrapper': {
+				'margin-bottom': '23px',
+			},
+			'.ugb-pricing-box__subprice': {
+				'margin-bottom': '28px',
+			},
+			'.ugb-pricing-box__button': {
+				'margin-bottom': '33px',
+			},
+			'.ugb-pricing-box__description': {
+				'margin-bottom': '38px',
+			},
+		} )
+
+	// Image Tab
+	cy.collapse( 'Image' )
+
+	desktopOnly( () => {
+		cy.adjust( 'Shape', {
+			label: 'Blob 1',
+			value: 'blob1',
+		} )
+		cy.adjust( 'Flip Shape Horizontally', true )
+		cy.adjust( 'Flip Shape Vertically', true )
+		cy.adjust( 'Stretch Shape Mask', true ).assertClassName( 'img.ugb-img--shape', 'ugb-image--shape-stretch' )
+
+		// We won't be able to assert image size for now since it requires server handling.
+	} )
+
+	cy.adjust( 'Image Width', 300, { viewport } )
+	cy.adjust( 'Force square image', true, { viewport } ).assertComputedStyle( {
+		'.ugb-img': {
+			'width': '300px',
+			'height': '300px',
+		},
+	} )
+	// TO DO: Make assert align for image
+
+	// Title, Price, Price Prefix, Price Suffix, Sub Price and Description Tabs
+	const typographyAssertions = [ 'title', 'price', 'price-prefix', 'price-suffix', 'subprice', 'description' ]
+	typographyAssertions.forEach( typographyAssertion => {
+		// Handle Sub Price label
+		const label = typographyAssertion === 'subprice' ? 'Sub Price' : startCase( typographyAssertion )
+
+		cy.collapse( label )
+
+		if ( typographyAssertion === 'title' ) {
+			desktopOnly( () => {
+				cy.adjust( 'Title HTML Tag', 'h4' ).assertHtmlTag( '.ugb-pricing-box__title', 'h4' )
+			} )
+		}
+
+		desktopOnly( () => {
+			const colorTitle = ( ! typographyAssertion.match( '/price/' ) ) ? label : 'Text'
+			cy.adjust( `${ colorTitle } Color`, '#742f2f' ).assertComputedStyle( {
+				[ `.ugb-pricing-box__${ typographyAssertion }` ]: {
+					'color': '#742f2f',
+				},
+			} )
+		} )
+
+		const textClass = typographyAssertion === 'price' ? 'price-line' : typographyAssertion
+		cy.adjust( 'Size', 55, { viewport, unit: 'px' } ).assertComputedStyle( {
+			[ `.ugb-pricing-box__${ textClass }` ]: {
+				'font-size': '55px',
+			},
+		} )
+		cy.adjust( 'Size', 2, { viewport, unit: 'em' } ).assertComputedStyle( {
+			[ `.ugb-pricing-box__${ textClass }` ]: {
+				'font-size': '2em',
+			},
+		} )
+
+		let typographySettings = {}
+		// Only for Price Prefix and Price Suffix
+		if ( typographyAssertion.match( '/price-/' ) ) {
+			typographySettings = {
```

Comment:
make the `typographySettings` constant

directly mutate the properties instead e.g.
```
object.property = 'changed value'
```